readme.md

```diff
@@ -197,6 +197,17 @@ XO is based on ESLint. This project started out as just a shareable ESLint confi
 - [Sublime Text](https://github.com/sindresorhus/SublimeLinter-contrib-xo)
 - [Atom](https://github.com/sindresorhus/atom-linter-xo)
 
+### Vim
+
+You can use [Syntastic](https://github.com/scrooloose/syntastic)'s ESLint checker with the following settings in your `.vimrc` file:
+
+```vim
+let g:syntastic_javascript_eslint_generic = 1
+let g:syntastic_javascript_eslint_exec = 'xo'
+let g:syntastic_javascript_eslint_args = '--compact'
+let g:syntastic_javascript_checkers = ['eslint']
+```
```

Comment:
Not sure that's something we want to support tbh. You're doing two things that are edge-casey and not really intended functionality: using `PATH` with relative paths, and then using syntastic's chaining ability to select the linter you want to use. While it works fine in _your_ environment that's not something I think will work globally in _everyone's_ environment.

For clarity, from the Syntastic docs:

> [Multiple checkers tells] syntastic to run the php checker first, and if no errors are found, run phpcs, and then phpmd[, etc.].